src/main/java/billgates/view/gui/ActionTextArea.java

```diff
@@ -0,0 +1,25 @@
+package billgates.view.gui;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ActionTextArea extends JTextArea {
+
+    public static final int DEFAULT_FONT_SIZE = 14;
+    public static final Font DEFAULT_FONT = new Font("Modern No. 20", Font.PLAIN, DEFAULT_FONT_SIZE);
```

Comment:
I'm seeing the `DEFAULT_FONT` a lot of times, do we want to consider saving it somewhere (Probably `config.properties` or somewhere) so that we can use it repeatedly elsewhere? 

Similar logic applies to the `DEFAULT_BACKGROUND_COLOR` thing

Reply:
There are multiple DEFAULT_FONT's, but with different values.